node/actors/sync_blocks/src/peers/tests/multiple_peers.rs

```diff
@@ -11,25 +12,26 @@ impl Test for RequestingBlocksFromTwoPeers {
 
     fn tweak_config(&self, config: &mut Config) {
         config.sleep_interval_for_get_block = BLOCK_SLEEP_INTERVAL;
-        config.max_concurrent_blocks = 2;
+        config.max_concurrent_blocks = 5;
         config.max_concurrent_blocks_per_peer = 1;
         // ^ Necessary for blocks numbers in tests to be deterministic
```

Comment:
yes, it is intentional. This way we put a bound on both traffic to peers and the cache size. It makes sense, because on average there is no point for fetching data faster than it can be stored. We need a buffer only because both storage speed and fetching speed may vary over time.